auth-api/src/auth_api/resources/affiliation_invitation.py

```diff
@@ -163,3 +163,64 @@ def put(affiliation_invitation_id, affiliation_invitation_token):
         except BusinessException as exception:
             response, status = {'code': exception.code, 'message': exception.message}, exception.status_code
         return response, status
+
+
+@cors_preflight('GET,DELETE,OPTIONS')
+@API.route('/<string:affiliation_invitation_id>/authorize',
```

Comment:
Is there a reason why we're building so many routes for affiliation invitations? Can we slim these down and reuse what Ody already built?

Reply:
I would prefer to have this as additional route as it allows me to add checks that might not be necessary for other routes. Like checking that person authorizing the invitation is actually part of the org.

Reply:
and why couldn't those conditionally be built into the route that exists?

Reply:
I am reusing some (like creation of invitation), but I think that some actions are better pulled out in separate endpoints (like authorizing/not authorizing) to keep other endpoints more simple. I do try to keep a balance between increasing complexity on certain endpoints and adding new ones.

Is there a specific issue with having more endpoints?

Reply:
Updated it. Now anything that does search can be done via GET on resource.